SnapWar/Themes/Font.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Fonts.swift
+//  SnapWar
+//
+//  Created by Pia Leung on 06/04/2020.
+//  Copyright © 2020 stickerwar. All rights reserved.
+//
+
+import Foundation
```

Comment:
we could probably reuse the functionalities in https://github.com/davidozhang/spycodes/blob/master/Spycodes/Core/SCFonts.swift and https://github.com/davidozhang/spycodes/blob/master/Spycodes/Core/Types/SCFontType.swift

Reply:
noted 👍